### PR TITLE
Implement calculator

### PR DESCRIPTION
**Pull request by JBsoftwire**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -5,11 +5,91 @@ namespace Calculator
...
+
+      int[] numbers = new int[count];
+      for (int index = 0; index < count; index++)
+      {
+        numbers[index] = AddNumber($"Please enter number {index + 1}: ");
+      }
+      return numbers;
+    }
+
+    private static int AddNumber(string message)
```
Comment (JBsoftwire) [inline Program.cs:55]: This is a confusingly named function in a calculator app, the natural expectation is that this function adds numbers together. Maybe 'AppendNumber'.

File: Program.cs [hunk 2]
```diff
@@ -5,11 +5,91 @@ namespace Calculator
...
+        }
+        else
+        {
+          if (op == "/")
+            answer = answer / number;
+          else if (op == "+")
+            answer = answer + number;
+          else if (op == "-")
+            answer = answer - number;
+          else
```
Comment (JBsoftwire) [inline Program.cs:86]: What about the "*" operator?

**Comment by catalin-ursachi** [comment]
Could you write all comments as a single review and submit them all together please? :)

File: Program.cs [hunk 3]
```diff
@@ -5,11 +5,91 @@ namespace Calculator
...
+      Console.Write("Please enter the operator: ");
+      string op = Console.ReadLine();
+      return op;
+    }
+
+    private static int[] AskForNumberArray(string op)
+    {
+      Console.Write($"How many numbers do you want to {op}? ");
+
+      int count = int.Parse(Console.ReadLine());
```
Comment (JBsoftwire) [inline Program.cs:45]: This only handles raw integers properly. What if they input a text string ('two') or something inappropriate ('help')?

File: Program.cs [hunk 4]
```diff
@@ -5,11 +5,91 @@ namespace Calculator
...
+      int number;
+
+      int.TryParse(Console.ReadLine(), out number);
+
+      return number;
+    }
+
+    public static int CalculateAnswer(string op, int[] numbers)
+    {
+      // Returns 0 if there aren't any numbers.
```
Comment (JBsoftwire) [inline Program.cs:68]: This comment is incorrect, if there's no numbers it returns an out of bounds error. Either the comment should be corrected or the empty array catch implemented.

**Comment by JackMeadDev** [comment]
Closing as the example PR has been successfully raised 😄

**Pull request closed without merging**